### ISSUE TITLE
new screenshots

### ISSUE THREAD

**Issue by fedelibre** [frescobaldi/frescobaldi#1211]
I can't remember where we've discussed this before.
I'm about to send a PR to get Frescobaldi 3.0.0 packaged as flatpak on Flathub. As I have to include some screenshots, I'd like to discuss it with you.

We should follow the tips you can find [in the Appstream spec](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html). In a nutshell, images should:

- be in PNG format
- width no smaller than 620 px
- 16:9 ratio
- use english language and default styles

IMO we should have only one image in full window and then use "zoomed" images to better show each feature.
The number of images is up to us.
See for example [MuseScore screenshots on Flathub](https://github.com/flathub/org.musescore.MuseScore/tree/master/screenshots).

@uliska Maybe you have some ready screenshots? Or more suggestions?
Let me know and then I'll start posting some screenshot here.

**Comment by uliska** [comment]
No, I don't have any that could be used for that purpose. They are mostly too specific, and recently with custom themes anyway.

**Comment by fedelibre** [comment]
Ok, eventually I found some time to make some screenshots.
To be used in the Appdata file (see #879) and in the flatpak package (see #1182 ).
Right now the target version is 3.0.0. We can update the images after 3.1.0 release.

The Appdata file should link to a repository hosting these images.
So the images should be added to a Frescobaldi repository before fixing issue 879.

Let me have your feedback and of course you are welcomed to send your screenshots.

![main-window](https://user-images.githubusercontent.com/1644647/69150986-10d89880-0ada-11ea-8e40-ed3f7a1ea00b.png)

![magnifier](https://user-images.githubusercontent.com/1644647/69151424-e89d6980-0ada-11ea-9ecd-1e62a088eab2.png)

![edit-in-place](https://user-images.githubusercontent.com/1644647/69151435-efc47780-0ada-11ea-917b-580abf4e8a99.png)

![autocompletion](https://user-images.githubusercontent.com/1644647/69151460-f7841c00-0ada-11ea-9577-1d2d358770ef.png)

![snippets](https://user-images.githubusercontent.com/1644647/69151479-01a61a80-0adb-11ea-8d27-9125dbd473b3.png)
![snippets-edit](https://user-images.githubusercontent.com/1644647/69151482-023eb100-0adb-11ea-9221-191052d5c92d.png)
![wizard-1](https://user-images.githubusercontent.com/1644647/69151502-08cd2880-0adb-11ea-9c17-92886ef0a846.png)
![wizard-2](https://user-images.githubusercontent.com/1644647/69151503-0965bf00-0adb-11ea-9ec4-82c868830ec7.png)
![wizard-3](https://user-images.githubusercontent.com/1644647/69151505-0965bf00-0adb-11ea-915e-9e8acac98a3b.png)

**Comment by uliska** [comment]
Nice!

One suggestion is to have something more "interesting" zoomed in with the magnifier, e.g. a dense engraving situation, with slur control points printed or something. 

I will (hopefully) provide a screenshot with the manuscript viewet.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by fedelibre** [comment]
> One suggestion is to have something more "interesting" zoomed in with the magnifier, e.g. a dense engraving situation, with slur control points printed or something.

Yes, I wanted to take a more dense engraving situation but I was lazy in searching good examples.
If you have any hint for scores in Mutopia (possibly >=2.18.2), let me know.

I took a screenshot of Paganini with control points:

![magnifier-control-points](https://user-images.githubusercontent.com/1644647/69188369-b280d980-0b1c-11ea-8f0e-e20b0f9f463d.png)

**Comment by uliska** [comment]
I'd say that's sufficient.

I'm thinking of doing something with "crazy" notstion as well, e.g. with my multi-segment slurs.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by fedelibre** [comment]
I decided I will send the PR to Flathub tonight without waiting for other feedback here. I want to get Frescobaldi 3.0.0 in Flathub asap. Then we'll have some time to adapt the master branch before 3.1.0 release.
The appdata file I'm going to submit to Flathub is [here](https://github.com/fedelibre/org.frescobaldi.Frescobaldi/blob/v3.0.0/org.frescobaldi.Frescobaldi.appdata.xml).

We can now consider also screenshots of new features such as the Extensions and the Document Fonts.

Where do you want to place these screenshot files? Should we create a directory in the root of this repository?

**Comment by uliska** [comment]
I would wait with the extensions stuff for 3.1

Right now there isn't anything really useful to show.
-- 
Diese Nachricht wurde von meinem Android-Gerät mit K-9 Mail gesendet.

**Comment by fedelibre** [comment]
Fixed by #1842
